couchbase.py

```diff
@@ -303,6 +304,20 @@ def _parse_metrics(obj_to_parse, dimensions, request_type, module_config):
                     if metric:
                         metrics.append(metric)
 
+        if 'hot_keys' in obj_to_parse:
+            hot_keys = obj_to_parse['hot_keys']
+            key = 0
+            metric_name_pref = 'bucket.hot_keys'
+            for keys in hot_keys:
+                for name, value in keys.items():
+                    if isinstance(value, numbers.Number):
```

Comment:
i usually use `name, value` when iterate into `dict` but ok i just changed to ops np